src/pages/UpdateServers.js

```diff
@@ -752,7 +795,14 @@ class UpdateServers extends BaseUpdateWizardPage {
 
   proceedOldComputeNotReachable = () => {
     this.setState({
-      showNoMigrationWarning: false, showReplaceModal: true});
+      showComputeNotReachableWarning: false, showReplaceModal: true});
+  }
+
+  cancelOldComputeNotReachable = () => {
+    this.setState({
```

Comment:
Changing three state variables to different values (some false, some undefined) just from cancelling one message suggests that there are too many intertwined states in this function. While I don't expect this to change now, please consider simplifying this if you have to make any substantial changes to this functionality.

Reply:
@GarySmith there a 2 situation to deal with has instances , has no instances and failed to get instances. When failed to get instances , that complicates this a little bit. I will see if there is way to simplify this.

Reply:
I just re-read the code, and there are only two variables (not three), so it's not really that bad. I don't want you to spend a lot of time reworking this, but, maybe this can be simplified (if possible) in a later PR if time permits.

Reply:
Ok good to know. I plan to come back to the UI code once I finished the playbook